cczoo/sgx_kms/include/key_exchange/se_cdefs.h

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (C) 2011-2021 Intel Corporation. All rights reserved.
```

Comment:
modify the copyright to GPL, like below:

#
# Copyright (c) 2021 Intel Corporation
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#    http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.

Reply:
have done.